[2021-03-23 20:43:19,139] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: EDGAR-Annotation-Pipeline.UploadLabledData 2021-03-24T03:15:58.972660+00:00 [queued]>
[2021-03-23 20:43:19,143] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: EDGAR-Annotation-Pipeline.UploadLabledData 2021-03-24T03:15:58.972660+00:00 [queued]>
[2021-03-23 20:43:19,144] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-23 20:43:19,144] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-23 20:43:19,144] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-23 20:43:19,150] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): UploadLabledData> on 2021-03-24T03:15:58.972660+00:00
[2021-03-23 20:43:19,152] {standard_task_runner.py:52} INFO - Started process 46015 to run task
[2021-03-23 20:43:19,159] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'EDGAR-Annotation-Pipeline', 'UploadLabledData', '2021-03-24T03:15:58.972660+00:00', '--job-id', '103', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/edgar_annotation.py', '--cfg-path', '/var/folders/_j/s8vbc0cs1fvgclq8hnvtq1wm0000gn/T/tmp_5lrxi_m', '--error-file', '/var/folders/_j/s8vbc0cs1fvgclq8hnvtq1wm0000gn/T/tmp2dm41xso']
[2021-03-23 20:43:19,160] {standard_task_runner.py:77} INFO - Job 103: Subtask UploadLabledData
[2021-03-23 20:43:19,192] {logging_mixin.py:104} INFO - Running <TaskInstance: EDGAR-Annotation-Pipeline.UploadLabledData 2021-03-24T03:15:58.972660+00:00 [running]> on host prathyushas-mbp.home
[2021-03-23 20:43:19,213] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=EDGAR-Annotation-Pipeline
AIRFLOW_CTX_TASK_ID=UploadLabledData
AIRFLOW_CTX_EXECUTION_DATE=2021-03-24T03:15:58.972660+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-24T03:15:58.972660+00:00
[2021-03-23 20:43:19,314] {credentials.py:1222} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2021-03-23 20:43:26,340] {logging_mixin.py:104} INFO - annotation file uploaded
[2021-03-23 20:43:26,343] {python.py:118} INFO - Done. Returned value was: None
[2021-03-23 20:43:26,347] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=EDGAR-Annotation-Pipeline, task_id=UploadLabledData, execution_date=20210324T031558, start_date=20210324T034319, end_date=20210324T034326
[2021-03-23 20:43:26,361] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-23 20:43:26,380] {local_task_job.py:146} INFO - Task exited with return code 0
